parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -243,7 +243,7 @@ installContainerd() {
 
 installImg() {
     img_filepath=/usr/local/bin/img
-    retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.5.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
```

Comment:
Does this have a CDN in front of it? I wonder if we should do that for the new url too...

Reply:
/hold

Yes acs-mirror.azureedge.net is a CDN setup, so we should probably do similarly for the new storage. No rush on this of course.

Reply:
ok cdn created - 

```
╰ wget https://upstreamartifacts.azureedge.net/img/img-linux-amd64-v0.5.6               ado-admin/default ⎈     03:57:03  
--2020-01-16 15:57:49--  https://upstreamartifacts.azureedge.net/img/img-linux-amd64-v0.5.6
Resolving upstreamartifacts.azureedge.net (upstreamartifacts.azureedge.net)... 13.107.246.10, 2620:1ec:bdf::10
Connecting to upstreamartifacts.azureedge.net (upstreamartifacts.azureedge.net)|13.107.246.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 25700016 (25M) [application/octet-stream]
Saving to: ‘img-linux-amd64-v0.5.6’

img-linux-amd64-v0.5.6           100%[=======================================================>]  24.51M  1.74MB/s    in 17s     

2020-01-16 15:58:08 (1.42 MB/s) - ‘img-linux-amd64-v0.5.6’ saved [25700016/25700016]
```

Reply:
Thanks @jchauncey! I updated to reference that CDN-fronted URL.

/hold cancel

Reply:
/hold cancel